### ISSUE TITLE
Help with getting reveal working with Angular

### ISSUE THREAD

**Issue by whittssg** [hakimel/reveal.js#1597]
Hello,

Can anyone point me in the right direction to getting reveal working with Angular?

I included the reveal.js and css file in my index file..

I then tried to use the barebones.html sample (removing the head body and script) that was provided and running Reveal.initialize(); in my controller.. I get nothing when i got to the view.. No console errors either.

I tried adding Reveal.initialize(); inside a doc ready:

 ```
  angular.element(document).ready(function () {
                Reveal.initialize();

            });
```

still nothing.. i also tried a time out with several different timeouts but i dont see anything.. Is there somthing simple that i am missing ?

Thanks

EDIT... i tried initializing it in my app.js like i do for other components as well but app wont even load when i tried that.

**Comment by tristansokol** [comment]
This would probably best be asked on [StackOverflow](http://stackoverflow.com/questions/tagged/reveal.js)

see
[https://github.com/hakimel/reveal.js/blob/master/CONTRIBUTING.md#personal-support](https://github.com/hakimel/reveal.js/blob/master/CONTRIBUTING.md#personal-support)

**Comment by demoneaux** [comment]
Yep, I don't have much experience with Angular, so asking on StackOverflow is probably the best idea.

**Comment by akash-pal** [comment]
For angularJS:

I actually commented out the below line:

  ```
   // Make sure we only initialize once
     // if( initialized === true ) return;

```
This enabled me to initialize again when I changed the page. 

I also had autoSlide on
     
       `Reveal.configure({ autoSlide: 4000 });`

So when I changed the page, I wrote a function to cancel the time and slide to the first slide. 
       
      ```
 cancelTimer: function(){
			pause();
			slide(0);
	}
```

I have then exposed the cancelTimer via the API.

This way I am able to use reveal in AngularJS

@bnjmnt4n I would like to know why reinitializing twice is not recommended?
